### ISSUE TITLE
Unable to access all of the menu's on the main Facebook Page with NVDA-2012.2.1

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2462]
**Reported by Brendon22 on 2012-06-18 05:35**
Hi, I'm Unable to access my Timeline editing menu's on the main Facebook Page. Such as removing posts and friend activities with NVDA-2012.2.1 and with the latest Snapshots? But I was able to before with version's before NVDA-2012.2.1? When pressing space or enter NVDA used to read us what was in the menu with out a problem. But now it doesn't?

**Comment by leonardder** [comment]
I'm able to do these things just fine. Furthermore, it is likely that many things have changed in Facebook, browsers and NVDA. Closing as works for me, but please get into contact if you think this is still not functioning as it should. IN that case, please make sure that the problem is NVDA related and not just Facebook being bad in a11y.
